dashboard/config/linux: add support for Android 6.1 and bump the other android tags

ambarus opened the pull request:
Add support for Android 6.1. Update configs to latest android branches.

Cc: joneslee@google.com

ambarus commented:
@dvyukov, @a-nogikh: Hi! Would you please consider these patches for integration?

a-nogikh reviewed (dismissed):
LGTM

dvyukov commented on dashboard/config/linux/bits/android-5.10.yml:
```diff
@@ -4,10 +4,10 @@
 # for details on how to checkout and build Android kernel.
 
-# Following branch: android12-5.10-lts
+# Following branch: android13-5.10-lts
```
How important is it to switch from android12 to android13? Do they have different kernels or configs?

Changing branch on the fly may affect some of the bisection and tree origin detection logic. But I am not sure right now how exactly.

dvyukov replied:
If the kernel is different, then we also need to switch internal syz-ci configs to android13. Otherwise they will still test android12.

ambarus replied:
android13 has more features and a higher EOL than android12, so yes, the kernels and configs may be different but still based on 5.10. See  https://source.android.com/docs/core/architecture/kernel/android-common#kmi-branch-lifecycle . I think it's better to test on the latest android version, but let's see what @lag-linaro thinks too.

lag-google replied:
It's okay to make the switch, but as Dmitry says, you need to update the configs as well.

Why don't you handle that as part of an othognal pull-request?

Let's only create `v6.1` in this one.

dvyukov replied:
> Why don't you handle that as part of an othognal pull-request?

syz-ci configs are stored elsewhere (internally)

lag-google replied:
Presumably the kernel configs will also need to be rebuilt too.

ambarus replied:
Indeed, it seems there's a difference between the dashboard's kernel configs and the  syz-ci kernel configs. Take https://syzkaller.appspot.com/bug?id=0f1e1644d7ed28d1d925d6c9e2e2c0c1c18cb794 for example, it uses a v5.10.177 kernel, while the kernel version specified by android-5.10.yml is 8d21bcc704ea1 (v5.10.82). Wondering why we define a kernel config for dashboards now.

dvyukov commented on dashboard/config/linux/android-5.4.config:
```diff
@@ -1331,7 +1341,54 @@ CONFIG_NET_FLOW_LIMIT=y
+# CAN Device Drivers
+#
+# CONFIG_CAN_VCAN is not set
+# CONFIG_CAN_VXCAN is not set
```
It can make sense to enable VCAN/VXCAN as well. I am not sure we have any of CAN devices otherwise and CAN stack won't be covered. But the virtual devices may to cover some of the CAN stack.

ambarus replied:
This could be done as an explicit separate patch. The config was generated with syz-kconf, there's nothing I added manually. I'll take a look into VCAN/VXCAN, thanks for the tip.

lag-google replied:
Right, let's keep it as vanilla as possible.  Subsequent config changes should be separate and obvious.

Not sure how relevant CAN devices are either.  We don't appear to have them enabled in ACK.

ambarus commented:
I've just forced push and updated the android tags to the latest SHA1s. More than that I introduced a new commit, "Revert "dashboard: bump CONFIG_FRAME_WARN for Android kernels"" as the build error reported at https://syzkaller.appspot.com/bug?id=0f1e1644d7ed28d1d925d6c9e2e2c0c1c18cb794 should now be fixed with the bump as well. More details in the commit subject. I don't have enough rights to see the syz-ci configs repo, so we have to either give me rights to change the Android 5.10 kernel, or to update the syz-ci configs by yourself. Thanks.
